### PR TITLE
Fix that thread problem with this simple hack

### PR DESCRIPTION
**Pull request by j574y923**

### CODE REVIEW

File: src/smp/components/staff/StaffInstrumentEventHandler_Hack.java [hunk 1]
```diff
@@ -120,7 +128,12 @@ public StaffInstrumentEventHandler_Hack(Staff s, ImageLoader i) {
         }
     }
 
-    private void disableAllStackPanes() {
+	/**
```
Comment (DC37) [inline src/smp/components/staff/StaffInstrumentEventHandler_Hack.java]: Nice, I like this

**Comment by DC37** [comment]
Thank you so much for helping with this!!!
I'll get onto the code review & merging and we can get this put together, probably as a new release.

**Comment by DC37** [comment]
Ok, I looked at it a little bit, there are still a few commented-out pieces of code that I'd like to delete, as well as that small merge conflict. Will test quite soon.

**Comment by j574y923** [comment]
aight cool

**Pull request closed without merging**